tests/testsuite/package.rs

```diff
@@ -6100,8 +6103,7 @@ fn registry_not_inferred_because_of_missing_option() {
         .masquerade_as_nightly_cargo(&["package-workspace"])
         .with_status(101)
         .with_stderr_data(str![[r#"
-[ERROR] `dep` cannot be packaged.
-The registry `crates-io` is not listed in the `package.publish` value in Cargo.toml.
+[ERROR] `package.publish` settings have no registry in common
```

Comment:
- We tend to refer to these as "fields" rather than "settings"
- This feels passive

What about

```
[ERROR] conflicts between `package,publish` fields between the selected packages
```